### PR TITLE
testimonial block

### PR DESCRIPTION
**Pull request by nicocapalbo**

### CODE REVIEW

File: pfdp/css/cards.css [hunk 1]
```diff
@@ -124,4 +157,12 @@
   .pill img {
     margin-right: 12px;
   }
+  
+  .testimonials {
+    display: grid;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/cards.css]: no need to repeat the values which stay the same. i think only `grid-template-columns changes` here

File: pfdp/homepage.html [hunk 2]
```diff
@@ -224,9 +234,76 @@ <h5 class>Remerciez vos donateurs</h5>
...
+          <div class="testimonial">
+            <img src="" alt="">
+          </div>
+          <div class="testimonial bg-yellow">
+            <div class="testimonial-text burgundy">
+              <p class="testimonial-title">Marie, 35 ans</p>
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Velit commodo egestas quam nulla facilisis nulla leo, neque. Tempor a bibendum fringilla.</p>
+            </div>
+          </div>
+          <div class="testimonial">
```
Comment (cecile-dzy-ncl) [inline pfdp/homepage.html:296]: why an empty testimonial ?

File: pfdp/css/cards.css (hunk 1 above)
Comment (nicocapalbo) [reply pfdp/css/cards.css]: removed the necessary values

File: pfdp/homepage.html (hunk 2 above)
Comment (nicocapalbo) [reply pfdp/homepage.html:296]: added images from Unsplash

**Review by cecile-dzy-ncl — APPROVED** [review]

**Pull request merged**